otp_patches/mnesia.erl.patch

```diff
@@ -0,0 +1,136 @@
+diff --git a/lib/mnesia/src/mnesia.erl b/lib/mnesia/src/mnesia.erl
```

Comment:
How stable is this? It seems like it could be a problem if it changes slightly, say from R20.1 to R20.2 or so?

Reply:
I checked with current OTP master, but it would perhaps be better to have subdirectories for different supported OTP versions.